src/CoreBundle/private/library/classes/TFullGroupTable/TFullGroupTable.class.php

```diff
@@ -415,6 +415,61 @@ private function getInlineFromAttributes(array $attributes)
         return $inline;
     }
 
+    private function getTableEditorPagedef($pagedef): string
```

Comment:
Please order these new methods so that they appear after the method in which they were called first.

Reply:
Done.

Reply:
Not yet done :-)

I meant to move all three methods under `_BuildFilterSection`, as they are called in that method.

Reply:
OK, now under ``_BuildFilterSection`` ;-)